test/com/amazon/ion/TimestampTest.java

```diff
@@ -30,6 +30,7 @@
 import static com.amazon.ion.Timestamp.Precision.SECOND;
 import static com.amazon.ion.Timestamp.Precision.YEAR;
 import static com.amazon.ion.impl._Private_Utils.UTC;
+import static junit.framework.TestCase.assertEquals;
```

Comment:
Looks like this import probably isn't needed?

Reply:
Gargh, how did that get there?